packages/flutter/test/material/date_range_picker_test.dart

```diff
@@ -1047,3 +1063,4 @@ class _RestorableDateRangePickerDialogTestWidgetState extends State<_RestorableD
     );
   }
 }
+
```

Comment:
This change is causing the analyzer test to fail, let's remove it.

```suggestion
```

Reply:
This should be fixed now.